spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -32,7 +32,16 @@
     end
   end
 
-  context 'when using Rails.root called by double quoted string' do
+  context 'when using Rails.root without path called by double quoted string' do
+    let(:source) { '"#{Rails.root}"' }
```

Comment:
I think we need more test case.
The issue ( #3947 ) problem is `"rm -rf #{Rails.root.join("foo")}"`, so, at a minimum, this should be tested.